cluster-scope/base/namespaces/democratic-csi/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: democratic-csi
+
+resources:
+  - namespace.yaml
+
+components:
+  - ../../../components/project-admin-rolebindings/operate-first
```

Comment:
hrmm this ocp group is a little misleading, let's remove this as this will give access to folks other than the cluster-admins

Reply:
Done.